doc/particles.md

```diff
@@ -305,14 +305,18 @@ class RectComponent extends Component {
 ```
 
 ## Flare Particle
-A container for `FlareAnimation`, it propagates `update` and `render` hooks to its child.
+
+To use Flare within Flame, use the [`flame_flare`](https://github.com/flame-engine/flame_flare) package.
+
+A container for `FlareActorAnimation`, it propagates `update` and `render` hooks to its child.
```

Comment:
nit: `It will provide a class `Flare Particle` that is a container...` might be a bit clearer. Since the introduction of the first paragraph I think this phrase was left in a confusing state